scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -32,6 +32,8 @@
   <div class="container-fluid" id="igvDiv" style="padding:5px; border:1px solid lightgray"></div>
 </body>
 
+{% set cors_anywhere_prefix = "https://cors-anywhere.herokuapp.com" -%}
```

Comment:
As long as the tracks are not secret in any way this looks ok! I guess we will depend on the proxy being up as well, but there are other dependencies as well. We could use this now, and keep looking for a local code solution perhaps?

Reply:
The page looks widely used but you are right, you never know! The good thing is that if it stops working then our igv browser won't crash, only the track will display error!